### DISCUSSION TITLE
Plotting and farming with lot's of used computers

### DISCUSSION THREAD

**Discussion by Lotsofusetdesktops** [Chia-Network/chia-blockchain#2702] [General]
Hi, 

My name is Jukka. I'm a used computer hole seller From Finland. I have never done anything with cryotos. But this couth my attention. All the sudden all my stock af used 2-6Tb SATA HDD:s was sold.
Low energy servers sales when up 1000%. I friend of mine who bought the last 20 2-4Gb SATA drive let me know what is going on. Thanks to Perttu I an VERY interested in chia. 

So. Here we are. I have 3 different machines making plots as a test. They produce 2 to 3 plots per day ea. Now it's time to scale up. I can put up 20-30-50 machines to temporary work for plotting and then move the plots to farmer machines. I have two empty rack cabinets to star with to long time farming.

So again. If there is some else who is going to this way I'm interested to open dialog and help other start as soon as I learn more.

My idea at the moment:
Multiple used computers to make plots. Just scale up, don’t use time to maximize single computer. Just ad more.
Use time to set up a wise farming server setup. 

Anyone trying to do it like this?

- Jukka

**Comment by love-chia** [comment]
呵呵，我以为只有我们这有二手电脑销售，原来各地都有。现在你们那儿的硬盘也涨价了吗？

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
